### PR TITLE
[TwigBridge][WebProfilerBundle] Require Twig 3.10

### PR DESCRIPTION
**Pull request by derrabus**
| Q             | A
| ------------- | ---
| Branch?       | 7.1
| Bug fix?      | no
| New feature?  | no
| Deprecations? | no
| Issues        | N/A
| License       | MIT

Requiring a more recent version of Twig allows us to drop some compat code.

### CODE REVIEW

File: src/Symfony/Bundle/WebProfilerBundle/Twig/WebProfilerExtension.php [hunk 1]
```diff
@@ -115,11 +115,6 @@ private static function escape(Environment $env, string $s): string
         }
 
         // Twig 3.9
-        if (method_exists(EscaperExtension::class, 'escape')) {
-            return EscaperExtension::escape($env, $s);
-        }
-
-        // to be removed when support for Twig 3 is dropped
-        return twig_escape_filter($env, $s);
+        return EscaperExtension::escape($env, $s);
```
Comment (fabpot) [inline src/Symfony/Bundle/WebProfilerBundle/Twig/WebProfilerExtension.php]: This is going to change in 3.10. So maybe we should go straight to 3.10 as a minimum version?

Comment (derrabus) [reply src/Symfony/Bundle/WebProfilerBundle/Twig/WebProfilerExtension.php]: I was afraid to do so because 3.10 is not released yet.

Comment (fabpot) [reply src/Symfony/Bundle/WebProfilerBundle/Twig/WebProfilerExtension.php]: 3.10 is now released :)

Comment (derrabus) [reply src/Symfony/Bundle/WebProfilerBundle/Twig/WebProfilerExtension.php]: Updated!

**Review by xabbuh — APPROVED** [review]

**Review by chalasr — APPROVED** [review]

**Comment by nicolas-grekas** [comment]
Thank you @derrabus.

**Pull request merged**